systemtest/src/test/java/io/strimzi/systemtest/kafka/listeners/ListenersST.java

```diff
@@ -668,6 +668,92 @@ void testLoadBalancerTls(ExtensionContext extensionContext) {
         );
     }
 
+    @ParallelNamespaceTest
+    @Tag(SANITY)
+    @Tag(ACCEPTANCE)
+    @Tag(EXTERNAL_CLIENTS_USED)
```

Comment:
I think this is not needed since we don't really use it as external listener here. We should base it on one of the internal listener tests I think. Or @im-konge

Reply:
If it isn't external listener and we can use the internal listener here, let's go with it.